### ISSUE TITLE
Gray out overloading methods

### ISSUE THREAD

**Issue by medoni** [tomasr/viasfora#286]
It would be really nice if overloading methods could be grayed out. For example, you have a bunch of ``Foo`` methods which doing only some parameter transformation.
```csharp
class MyClass {

    object Foo() {
        return Foo(false);
    }
    object Foo(bool b) {
        return FooCore(...);
    }
    object Foo(string s) {
        if (s == null) throw new ArgumentNullException(nameof(s));
        return FooCore(...);
    }

    object FooCore(...) {
        ...
  }
}
```

Current:
![image](https://user-images.githubusercontent.com/1647126/76495277-d1420d80-6436-11ea-9e0d-f344d4a04a50.png)

Expected:
![image](https://user-images.githubusercontent.com/1647126/76495776-df445e00-6437-11ea-81f9-0c1d193a4f64.png)

Your focus gets automatically to ``FooCore``

**Comment by tomasr** [comment]
It's an interesting proposal, but unfortunately not something that I can do in Viasfora, since it only relies on lexical analysis and not full syntax parsing.

It would likely be doable on a plugin based around Roslyn, though.

**Comment by medoni** [comment]
Implemented in PR #287
